Allows it to print clean output with our mksh-static.

--- a/neofetch
+++ b/neofetch
@@ -1640,7 +1640,7 @@ get_shell() {
 
         sh|ash|dash|es) ;;
 
-        *ksh)
+        *ksh|*ksh.static)
             shell+=$("$SHELL" -c "printf %s \"\$KSH_VERSION\"")
             shell=${shell/ * KSH}
             shell=${shell/version}
